test/dotnet-new3.UnitTests/Approvals/DotnetNewHelp.CanShowHelp.approved.txt

```diff
@@ -2,16 +2,20 @@
   Template Instantiation Commands for .NET CLI.
 
 Usage:
-  new3 [<template-short-name> [<template-args>...]] [options]
```

Comment:
check this

Reply:
interesting that on this example there are options (the global options, in this case help) but there's no `[options]` placeholder in the example usage.

Reply:
this is custom usage, likely a bug

Reply:
decided to do custom usage here as `NewCommand` doesn't have any visible options until reparsing.

Reply:
@baronfel could you please check the overall feel of this help?

`[options]` is a bit confusing for me because in the list below there is only help. 
however there are more options on template instantiation level (`--language`, `--type` etc) and also on subcommand level.